src/wallet/rpcdisclosure.cpp

```diff
@@ -59,10 +59,10 @@ UniValue z_getpaymentdisclosure(const UniValue& params, bool fHelp)
             "3. \"output_index\"    (string, required) \n"
             "4. \"message\"         (string, optional) \n"
             "\nResult:\n"
-            "\"paymentblob\"        (string) Hex string of payment blob\n"
+            "\"paymentdisclosure\"  (string) Hex data string, with \"zpd:\" prefix.\n"
             "\nExamples:\n"
-            + HelpExampleCli("z_getpaymentdisclosure", "96f12882450429324d5f3b48630e3168220e49ab7b0f066e5c2935a6b88bb0f2 0 0 \"refund\"")
-            + HelpExampleRpc("z_getpaymentdisclosure", "\"96f12882450429324d5f3b48630e3168220e49ab7b0f066e5c2935a6b88bb0f2\", 0, 0, \"refund\"")
+            + HelpExampleCli("z_getpaymentdisclosure", "zpd:96f12882450429324d5f3b48630e3168220e49ab7b0f066e5c2935a6b88bb0f2 0 0 \"refund\"")
+            + HelpExampleRpc("z_getpaymentdisclosure", "\"zpd:96f12882450429324d5f3b48630e3168220e49ab7b0f066e5c2935a6b88bb0f2\", 0, 0, \"refund\"")
```

Comment:
These are transaction IDs, not payment disclosures, so they shouldn't have prefixes.

Reply:
Good spot.  Fixed.

Reply:
You added the `zpd:`, but it's still followed by hex encoding instead of Base64 (or should it be Base58Check? cf. https://github.com/zcash/zcash/issues/2746#issuecomment-349943450).

Reply:
Transaction IDs are normally encoded in hex.